src/components/Slider.js

```diff
@@ -45,7 +53,7 @@ const Slider = () => {
             <div className="product-description">
               <h5>{title}</h5>
               <h4>{price}</h4>
-              <Link to={`/item/${id}`}>
+              <Link to={`/item/${id}`} onClick={productCountReSetHandler(id)}>
```

Comment:
why we should reset this? I am clicking to see product detail, there is no need to reset the product count in my card.